tests/net/lib/coap/src/main.c

```diff
@@ -23,6 +23,7 @@
 
```

Comment:
The commit subject could be make shorter, "tests: net: coap: ...." is enough. The commit subject is not meant to describe directory structure of the code anyway.

Reply:
Now the subject went too short, please set the subject to "**tests: net: coap: Added new changes to improve code coverage**"